src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Modifier as PopperModifier, Options } from '@popperjs/core';
+import classnames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from '../messages';
+import PopupBase from '../PopupBase';
+
+import './PopupList.scss';
+
+export type Modifier = PopperModifier<string, Partial<Options>>;
+
+export type Props = {
+    className?: string;
+    reference: HTMLElement;
+    modifiers: [Modifier];
```

Comment:
Should this be `Modifier[]`? Or is it intended to be a Tuple type? https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple